packages/apps/human-app/frontend/src/smart-contracts/get-network-name.ts

```diff
@@ -1,9 +1,15 @@
-import { MainnetChains, TestnetChains } from '@/smart-contracts/chains';
+import { AllTestnetsChains, AllMainnetChains } from '@/smart-contracts/chains';
 import { env } from '@/shared/env';
 
-export const getNetworkName = (): string => {
+export const getNetworkName = (
+  chainIdsEnabled: number[] | undefined,
+  chainId: number
+): string | null => {
+  if (!chainIdsEnabled) {
```

Comment:
No need to do this check here, it should always just try to map chain name.

When returning jobs for oracle we might display some from chains that are not in `chainIdsEnabled`, but you shouldn't worry about it: we should fix it on our end

Reply:
Good catch! I improved this function when I found a bug and changed the `arr` from which I am looking for the name. Because of this if statement I didn't realize that `chainIdsEnabled` is no longer needed, thanks :)